target/linux/generic/pending-5.15/350-mips-bmips-BCM6358-disable-RAC-flush-for-TP1.patch

```diff
@@ -135,39 +135,39 @@ Signed-off-by: Álvaro Fernández Rojas <noltari@gmail.com>
 
 --- a/arch/mips/bmips/dma.c
 +++ b/arch/mips/bmips/dma.c
-@@ -64,6 +64,8 @@ phys_addr_t dma_to_phys(struct device *d
- 	return dma_addr;
- }
+@@ -66,6 +66,8 @@ phys_addr_t dma_to_phys(struct device *d
+ 
+ bool bmips_rac_flush_disable;
  
 +bool bmips_rac_flush_disable;
 +
  void arch_sync_dma_for_cpu_all(void)
  {
  	void __iomem *cbr = BMIPS_GET_CBR();
-@@ -74,6 +76,9 @@ void arch_sync_dma_for_cpu_all(void)
- 	    boot_cpu_type() != CPU_BMIPS4380)
+@@ -77,6 +79,9 @@ void arch_sync_dma_for_cpu_all(void)
  		return;
  
-+	if (unlikely(bmips_rac_flush_disable))
+ 	if (unlikely(bmips_rac_flush_disable))
 +		return;
 +
++	if (unlikely(bmips_rac_flush_disable))
```

Comment:
This patch can be removed.
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/arch/mips/bmips?h=v5.15.106&id=65b723644294f1d79770704162c0e8d1f700b6f1

Reply:
You are correct, thanks.